src/components/featuredCategories/featuredCategories.js

```diff
@@ -0,0 +1,220 @@
+    <div className="feat-heading"> <h2>Featured Categories</h2></div>
+    <div className="feature-container">
+      <a href="https://github.com/JuvitaSaini09/">
+        <div className="card-featured">
+          <div className="card6 card">
+            <div className="card-badge">
+              <button className="heart-badge "><span className="heart">♡</span></button>
+              <img className="card-image" alt="card" src={atomicHabits} />
+              <div className="card-text card6-text">
+                <header>
+                  <h3>Atomic Habits</h3><button className="rate-btn"><span>★</span>4.5</button>
+                </header>
+                <div className="card-info">
+                  <p>Atomic Habits</p>
+                  <div className="price">₹. 300
+                    <span className="price-old">₹ 600</span>
+                  </div>
+                </div>
+              </div>
+              <footer>
+                <button className="btn-cart">Add to cart</button>
+              </footer>
+            </div>
+          </div>
+        </div>
+      </a>
+      <a href="https://github.com/JuvitaSaini09/">
+        <div className="card-featured">
+          <div className="card6 card">
+            <div className="card-badge">
+              <button className="heart-badge "><span className="heart">♡</span></button>
+              <img className="card-image" alt="card" src={ikigai} />
+              <div className="card-text card6-text">
+                <header>
+                  <h3>Ikigai</h3><button className="rate-btn"><span>★</span>4.5</button>
+                </header>
+                <div className="card-info">
+                  <p>Ikigai</p>
+                  <div className="price">₹. 300
+                    <span className="price-old">₹. 600</span>
+                  </div>
+                </div>
+              </div>
+              <footer>
+                <button className="btn-cart">Add to cart</button>
+              </footer>
+            </div>
+          </div>
+        </div>
+      </a>
+      <a href="https://github.com/JuvitaSaini09/">
+        <div className="card-featured">
+          <div className="card6 card">
+            <div className="card-badge">
+              <button className="heart-badge "><span className="heart">♡</span></button>
+              <img className="card-image" alt="card" src={alchemist} />
+              <div className="card-text card6-text">
+                <header>
+                  <h3>Alchemist</h3><button className="rate-btn"><span>★</span>4.5</button>
+                </header>
+                <div className="card-info">
+                  <p>Alchemist</p>
+                  <div className="price">₹. 300
+                    <span className="price-old">₹. 600</span>
+                  </div>
+                </div>
+              </div>
+              <footer>
+                <button className="btn-cart">Add to cart</button>
```

Comment:
Suggestion: here you are repeating product card with same elements, instead of this you can make component and use them

Reply:
Thankyou for the review ,i have updated it and added map to display cards